app/src/main/java/no/usn/mob3000/ui/screens/auth/ChangeEmailScreen.kt

```diff
@@ -83,6 +84,7 @@ fun ChangeEmailScreen(
                     onValueChange = { email = it },
                     label = { Text(stringResource(R.string.auth_email_input)) },
                     placeholder = { Text(stringResource(R.string.auth_login_email_placeholder)) },
+                    textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground),
```

Comment:
[See this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847123849).